plugins/dstat_nvidia_gpu.py

```diff
@@ -0,0 +1,41 @@
+### Author: Vasilis Vryniotis <bbriniotis@datumbox.com>
+
+class dstat_plugin(dstat):
+    """
+    Total GPU usage for NVIDIA cards. Requires the nvidia-ml-py package.
+
+    Usage: dstat --nvidia-gpu
+    """
+
+    def __init__(self):
+        self.name = 'gpu nvidia'
+        self.nick = ('gpu',)
+        self.vars = ('gpu',)
+        self.type = 'p'
+        self.width = 5
+        self.scale = 1
+
+    def check(self):
+        import pynvml
+        pynvml.nvmlInit()
+
+    def extract(self):
+        self.val['gpu'] = self._getTotalUsage(10)
+
+    def _getUsagePerGPU(self, samples):
```

Comment:
Yeah, happy to do it. Can you point me to the snippet that does the same for the CPU? The base dstat class does not provide comments for how the various internal fields work (vars, type, scale etc) so I'm not sure in which format I should return the results.